src/content/docs/en/release/41-upgrades.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Early Upgrades for Ultramarine 41
+description: "A temporary notice for the 39 to 40 upgrade path."
+---
+
+Like with Ultramarine 40, we're offering early upgrades from Ultramarine 40 to 41 ahead of image release.
+
+Fedora 41 is out, and Ultramarine 41's features are largely focused around the installation experience. Features outside of these are stable enough for most people to upgrade
+
+## Upgrading
+
+Open your Edition's app store, on Flagship, GNOME, and Xfce, this is Software, on Plasma this is Discover. The upgrade should be in the updates section, simply press upgrade, wait for the packages to download, and reboot from the app store.
+
+You may need to update your copy of Ultramarine 40 before the upgrade will become available.
```

Comment:
Talk about/guide for UMhop/Pantheon upgrade path?

Reply:
we can make another page for that

Reply:
That works